app/routes/payment-requests.js

```diff
@@ -0,0 +1,26 @@
+const Boom = require('@hapi/boom')
+const Joi = require('joi')
+const { resetPaymentRequestByInvoiceNumber } = require('../reset')
+
+module.exports = [{
+  method: 'POST',
+  path: '/payment-request/reset',
+  options: {
+    validate: {
+      payload: Joi.object({
+        invoiceNumber: Joi.string().required()
+      }),
+      failAction: async (request, h, error) => {
+        return Boom.badRequest(error)
```

Comment:
It might just be my destiny to use this HTTP error code: https://hapi.dev/module/boom/api/?v=9.1.4#boomteapotmessage-data